model-optimizer/extensions/front/tf/sparse_to_dense.py

```diff
@@ -0,0 +1,67 @@
+"""
+ Copyright (C) 2020 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import numpy as np
+
+from extensions.ops.Cast import Cast
+from extensions.ops.scatternd import ScatterNDUpdate
+from mo.front.common.replacement import FrontReplacementOp
+from mo.graph.graph import Node, Graph, rename_nodes
+from mo.ops.broadcast import Broadcast
+from mo.ops.const import Const
+
+
+class SparseToDense(FrontReplacementOp):
+    """
+    This replacer substitutes TensorFlow SparseToDense operation with Broadcast -> ScatterND chain.
+    The Broadcast operation creates a tensor filled with default value and of required shape.
+    The ScatterND operation updates the created tensor with required values at required locations.
+    """
+    op = "SparseToDense"
+    enabled = True
+
+    def run_after(self):
+        from extensions.front.tf.CTCGreedyDecoder import CTCGreedyDecoderReplacement
+        return [CTCGreedyDecoderReplacement]
+
+    def replace_op(self, graph: Graph, node: Node):
+        node_name = node.soft_get('name', node.id)
+
+        # broadcast default value to required shape
+        broadcast_node = Broadcast(graph, {'name': node_name + '/Broadcast_'}).create_node()
+        node.in_port(1).get_connection().set_destination(broadcast_node.in_port(1))
+        if not node.in_port(3).disconnected():
+            # cast default value
```

Comment:
Update a comment with description why this Cast is needed.

Reply:
Added an explanation that is due to W&D specific and OpenVINO limitation to support I64 input..